cmd/snap-confine/snap-confine.c

```diff
@@ -192,6 +192,8 @@ int main(int argc, char **argv)
 			sc_maybe_fixup_permissions();
 			sc_unlock(snap_name, snap_lock_fd);
 
+                        sc_setup_user_mounts(snap_name);
```

Comment:
Do you think we should setup user mounts for snaps running as root? That would easily solve the udisks problem.

Reply:
At the moment in my branch they're being set up as root, but it will need to change to regular user + CAP_SYS_ADMIN in order to work with FUSE file systems.  As mentioned earlier, I don't think the udisks2 snap will be affected one way or the other.